cob_bringup/robots/cob4-10.xml

```diff
@@ -35,15 +35,15 @@
 				<arg name="robot" value="$(arg robot)"/>
 				<arg name="pc" value="$(arg cob4-10-b1)"/>
 			</include>
-
-			<include file="$(find cob_bringup)/drivers/sick_flexisoft.launch">
-				<arg name="robot" value="$(arg robot)"/>
-			</include>
-			<include file="$(find cob_bringup)/drivers/bms.launch">
-				<arg name="robot" value="$(arg robot)"/>
-			</include>
 		</group>
 
+		<include file="$(find cob_bringup)/drivers/sick_flexisoft.launch">
+			<arg name="robot" value="$(arg robot)"/>
+			<arg name="sim" value="$(arg sim)"/>
+		</include>
+		<include file="$(find cob_bringup)/drivers/bms.launch">
+			<arg name="robot" value="$(arg robot)"/>
+		</include>
```

Comment:
Like this?
`<node pkg="rostopic" type="rostopic" args="pub <topic_name> <topic_type> <message_content> -r<rate>" name="<name>" output="screen"/>`
Could you put that into the bms.launch then...for now? Together with a comment in https://github.com/ipa320/msh/issues/586